### PR TITLE
ROX-18757: Update GetReportHistory url path

### PR DESCRIPTION
**Pull request by md2119**
## Description

Rename "/v2/reports/history/{report_config_id}" to "/v2/reports/configurations/{id}/history". Grepped the UI code to replace the usages.

## Checklist
- [ ] Investigated and inspected CI test results
- ~Unit test and regression tests added~
- ~Evaluated and added CHANGELOG entry if required~
- ~Determined and documented upgrade steps~
- ~Documented user facing changes~

If any of these don't apply, please comment below.

## Testing Performed
Unit

image

### CODE REVIEW

File: proto/api/v2/report_service.proto
Comment (charmik-redhat) [inline proto/api/v2/report_service.proto:101]: Minor : Should we move this to `proto/api/v2/report_configuration_service.proto` since it is based on configuration now? 

I am perfectly fine keeping it here too.

**Review by charmik-redhat — APPROVED** [review]
BE changes lgtm , @sachaudh handing over to you for FE change.

File: proto/api/v2/report_service.proto
Comment (md2119) [reply proto/api/v2/report_service.proto:101]: Good point. All report APIs should be provided under one service. There are two services catering the reporting feature, and there is not need to have them separate especially when they are tightly coupled. Once we align all the APIs with the guidelines, we should consolidate the two, hopefully soon.

**Comment by md2119** [comment]
The evidence that the updated API does not cause any regression in the UI can be found in PR description. In the interest of time, I cannot wait for @sachaudh's approval.

**Pull request merged**
